libs/application/templates/parental-leave/README.md

```diff
@@ -1,5 +1,138 @@
+
+Below you'll find a flow chart for the application
+
+[Flow chart]
+
+### Glossary
+
+| Word             | Description                   |
+| ---------------- | ----------------------------- |
+| Primary parent   | The mother carrying the child |
+| Secondary parent | The other parent              |
+| VMST             | Vinnumálastofnun              |
+
+### States
+
+This section assumes that you are familiar with [states](https://docs.devland.is/libs/application/core#states).
+
+#### Prerequisites
+
+This state is a temporary state that all new applications will be created in. It has a short lifespan and is unlisted.
+
+The purpose of this state is to be a guard into the actual application. To advance to the next state, an applicant will need to be expecting a child. To verify this, there is an external data step which fetches data from VMST and Þjóðskrá.
+
+#### Draft
+
+Valid applicants will be able to advance to this state where they can start the actual application and fill in all the relevant data.
+
+#### Other parent approval
+
+When the primary parent requests transferal of rights, then the other secondary parent will need to approve of the transfer. The other parent will receive an email with a link to approve of this request.
+
+#### Employer waiting to assign
+
+If the applicant is employed by an employer (not self employed), then they are asked to provide an email address which a confirmation email will be sent to. When the employer receives the email they can click a link which automatically assigns them to the application and advances it to the next state
+
+#### Employer approval
+
+Here the employer will have a chance to review the periods selected by the applicant as well as pension fund and union.
+
+The employer can then approve or request changes
+
+#### Vinnumalastofnun approval
+
+Applications that have been sent to VMST
+
+### Parental leave template API module
+
+For async actions that require server side logic we have the template api modules. The module for this application contains email templates as well as external actions like `sendApplication`.
+
+Have a look at `libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.module.ts` for further information on these actions, and `libs/application/template-api-modules/README.md` to get up to speed on template api modules.
+
+### API and X-Road
+
+All user interaction goes through our GraphQL API (`api`) which then handles communicating with other APIs.
+
+To communicate with VMST a request has to go through [X-Road](https://docs.devland.is/technical-overview/x-road). Both in development and on production environments.
+
+To connect to VMST test API you'll want to start the local proxy, following these steps:
+
+- Open AWS: https://island-is.awsapps.com/start#/
+  - island-is-development01 > AWSPowerUserAccess > Command line access
+- Choose an option you prefer in the modal
+- Connect to the proxy `./scripts/run-xroad-proxy.sh`
```

Comment:
This is kind of documented [here](https://docs.devland.is/#running-proxy-against-development-service) and [here](https://docs.devland.is/repository/aws-secrets#using-aws-session).